src/dotnet/commands/dotnet-restore/LocalizableStrings.resx

```diff
@@ -172,4 +172,19 @@ This is equivalent to deleting project.assets.json.</value>
   <data name="CmdInteractiveRestoreOptionDescription" xml:space="preserve">
     <value>Allows the command to stop and wait for user input or action (for example to complete authentication).</value>
   </data>
+  <data name="CmdLockedModeOptionDescription" xml:space="preserve">
+    <value>Do not allow updating project lock file.</value>
+  </data>
+  <data name="CmdLockFilePathOption" xml:space="preserve">
+    <value>ProjectLockFile_Path</value>
```

Comment:
I believe we used all CAPS for these values and all words are _ separated

Reply:
Is the file path required? If so, is the error where it is not provided handled by NuGet? And if so, should we just collapse the file path with say, use-lock-file?

Reply:
If not, could we describe briefly the default location where the lock file would be generated?

Reply:
It felt weird adding `PROJECT_LOCK_FILE_PATH`... how about `LOCK_FILE`

This is optional parameter, by default it creates ` project.lock.json` file in the project root directory which you can overwrite with this parameter. Do we specify default value for all other parameters?

Reply:
Not LOCK_FILE, that does not indicate path. `LOCK_FILE_PATH` seems good. Or, if this only appears in help, just `PATH` might be OK.

Livar can you answer the default question?

Reply:
I don't understand @jainaashish's question. What do you mean? Also, somewhere in the help description, I would say that if the path is not specified, we default to the project's folder. Don't know if that's what you were asking.

Reply:
I'll change it to `LOCK_FILE_PATH`